### PR TITLE
Adds status flag for monitors.

### PR DESCRIPTION
**Pull request by pingers**
Adds a status flag to monitors controlled from route annotation.

### CODE REVIEW

File: cmd/openshift-newrelic-synthetics/sync/command.go [hunk 1]
```diff
@@ -84,19 +90,19 @@ func syncSynthetics(client *newrelic.NewRelic, routes []routev1.Route, location
 
 		tags[m.Name] = []entities.Tag{
 			{
-				Key: entityutils.TagOpenShiftRouteNamespace,
+				Key:    entityutils.TagOpenShiftRouteNamespace,
```
Comment (behzad1q) [inline cmd/openshift-newrelic-synthetics/sync/command.go:93]: are the extra spaces intentional?

File: internal/openshift/route/const.go [hunk 2]
```diff
@@ -3,4 +3,6 @@ package route
 const (
 	// AnnotationIPWhitelist used when for skipping routes.
 	AnnotationIPWhitelist = "haproxy.router.openshift.io/ip_whitelist"
+	// NewRelicStatus used for toggling on/off New Relic Synthetics monitor.
+	NewRelicStatus = "one.newrelic.com/synthetics-status"
```
Comment (behzad1q) [inline internal/openshift/route/const.go:7]: NewRelicStatusUrl sounds better descriptive variable name.

File: cmd/openshift-newrelic-synthetics/sync/command.go (hunk 1 above)
Comment (pingers) [reply cmd/openshift-newrelic-synthetics/sync/command.go:93]: Yeah... go fmt enforces them... no choice on my part.

File: internal/openshift/route/const.go (hunk 2 above)
Comment (pingers) [reply internal/openshift/route/const.go:7]: It's not a url.. it's a qualifier for an annotation. Can see how it's confusing.

**Pull request merged**